src/migration/migrator.ts

```diff
@@ -461,27 +510,31 @@ export class Migrator {
       .withPlugin(this.#schemaPlugin)
       .selectFrom(this.#migrationTable)
       .select('name')
-      .orderBy('name')
+      .orderBy('timestamp')
```

Comment:
This can fail if two migrations have been run within the same millisecond. Either make sure each migration has a different timestamp or order by name after ordering by timestmp.

Reply:
Doesn't ordering by timestamp mean that if you first run migrations

```
1, 2, 4
```

and then

```
3
```

after that, migrating down will run the down migration of `3` 

producing

```
1, 2, 4
```

Is this really expected?

Reply:
Actually, secondary order by name probably covers all realistic cases 🤔

Reply:
Ok, I added a secondary sort by name. Thanks for the suggestion!